.github/workflows/deploy.yml

```diff
@@ -6,8 +6,12 @@ jobs:
     name: Deploy
     runs-on: ubuntu-latest
     steps:
-      - name: Dump event
-        run: jq . $GITHUB_EVENT_PATH
+      - name: Echo the client name
+        run: echo {{github.event.client_payload.repo}}
+        if: github.event.action == 'build_application'
+
+      - name: Set IP/domain for the server
+        run: echo "::set-env name=SERVER_IP::18.217.253.44"
```

Comment:
What does that `::` do? I think it might be better to make use of GitHub Actions' official way of setting environment variables: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables

Reply:
I was using another suggested github actions way: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#set-an-environment-variable-set-env